tools/cli/src/build.rs

```diff
@@ -59,12 +59,13 @@ pub(crate) fn build(trailing_args: Vec<&str>) -> CLIResult<()> {
                 .filter(|name| name.ends_with(".wasm"))
                 .collect::<Vec<_>>();
             if !new_wasms.is_empty() {
-                let sdk_version = extract_sdk_version(&manifest_path)?;
-                wasms.extend(
-                    new_wasms
-                        .into_iter()
-                        .map(|name| (name, sdk_version.clone())),
-                )
+                if let Ok(sdk_version) = extract_sdk_version(&manifest_path) {
```

Comment:
What about writing some sort of log message?

Reply:
I guess we can just add more extencive logging in marine cli. Like, log actions "skipped", "added IT", and so on. I dont think it is good to do this pr.

I was thinking about writing a log message in case where no wasms were modified, but looked at the comment at line 74 and decided that this is not needed:
`// it is possible to build a object file without Wasm artifacts`